.gitignore

```diff
@@ -31,4 +31,5 @@ tmp/
 pkg/
 dist/
 node_modules/
-.parcel-cache/
\ No newline at end of file
+.parcel-cache/
+wen/node_modules
```

Comment:
Is this change necessary?
I don't think we have `wen` folder in a project structure.

Reply:
How strange. That's meant to be web/node_modules. I'll update. Might not be necessary but just added to make sure it doesn't clog up the repo.